docs/editing.md

```diff
@@ -81,9 +81,9 @@ Pre-publication steps:
 - Check the archive deposit has the correct metadata (title and author list), and request the author edit it if it doesn’t match the paper.
 - Run `@whedon set <doi> as archive`.
 - Run `@whedon set <v1.x.x> as version` if the version was updated.
-- Ping the `@openjournals/joss-eics` team on the review thread letting them know the paper is ready for final processing.
+- Run `@whedon accept` to generate the final proofs and have Whedon notify the `@openjournals/joss-eics` team that the paper is ready for final processing.
```

Comment:
instead of "and have Whedon notify..." perhaps it should be ", which has Whedon notifiy..."